### ISSUE TITLE
Can we remove the protocol from apiBase

### ISSUE THREAD

**Issue by zstadler** [IsraelHikingMap/Site#150]
The code for embedding an Israel Hiking map in another site is of the form
```html
<iframe src='http://israelhiking.osm.org.il/#/?s=y2yrmJdWwF'
  width='800' height='600' frameborder='0' scrolling='no' />
```
As a result, it cannot be embedded in an HTTPS-based site without causing a "mixed contents" warning. I suggest the following code instead
```html
<iframe src='//israelhiking.osm.org.il/#/?s=y2yrmJdWwF'
  width='800' height='600' frameborder='0' scrolling='no' />
```

This embedding code uses apiBase as defined at IsrealHiking.Web/common/Urls.ts:
```javascript
public static apiBase = window.location.protocol + "//" + window.location.host + "/";
```

Can we remove ```window.location.protocol``` from the expression? 
I don't think it is needed, but if it will be needed at specific places it can be added there.

**Comment by HarelM** [comment]
I don't think it will be wise.
I think the mix mode warning is not coming from this code though - the tiles are fetched from http://... and not from https://...
I'll see if I can fix that.
I can also add a button in the share dialog with radio for http and https...?

**Comment by HarelM** [comment]
Related to #148.

**Comment by zstadler** [comment]
Clarification: This question is about mixed contents in other sites that embed Israel Hiking maps.
The mixed content warning issues in the Israel Hiking site were already fixed as part of #148.

Can you elaborate on the potential issues with this approach?

**Comment by HarelM** [comment]
Ignore my last comment, if the above address is a valid address then I see no issue here.
